Start errors

**What's happening?**
Trying to install this bot on my raspberry pi server.

**Logs**

<!-- Paste any logs here (between the backticks). -->
<!-- You can get extended logs by setting the environment variable DEBUG to *. -->
<!-- For example, DEBUG=* yarn start if you're running directly from the cloned repository. -->
```
> muse@1.8.2 start
> npm run env:set-database-url -- esmo src/scripts/migrate-and-start.ts


> muse@1.8.2 env:set-database-url
> esmo src/scripts/run-with-database-url.ts "esmo" "src/scripts/migrate-and-start.ts"

(node:21536) ExperimentalWarning: --experimental-loader is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
(node:21536) DeprecationWarning: Obsolete loader hook(s) supplied and will be ignored: getFormat, transformSource
(node:21574) ExperimentalWarning: --experimental-loader is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
(node:21574) DeprecationWarning: Obsolete loader hook(s) supplied and will be ignored: getFormat, transformSource
▲ [WARNING] Use "node_modules/youtube.ts/dist/Youtube.js" instead of "node_modules/youtube.ts/dist/youtube.js" to avoid issues with case-sensitive file systems

    <stdin>:1:7:
      1 │ import "youtube.ts/dist/youtube.js"
        ╵        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node:internal/errors:464
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/opt/muse/node_modules/youtube.ts/dist/youtube.js' imported from /opt/muse/src/services/third-party.ts
    at new NodeError (node:internal/errors:371:5)
    at finalizeResolution (node:internal/modules/esm/resolve:418:11)
    at moduleResolve (node:internal/modules/esm/resolve:983:10)
    at defaultResolve (node:internal/modules/esm/resolve:1080:11)
    at resolve (file:///opt/muse/node_modules/esbuild-node-loader/loader.mjs:126:12)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async ESMLoader.resolve (node:internal/modules/esm/loader:530:24)
    at async ESMLoader.getModuleJob (node:internal/modules/esm/loader:251:7)
    at async ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:81:21)
    at async Promise.all (index 2) {
  code: 'ERR_MODULE_NOT_FOUND'
}
file:///opt/muse/node_modules/execa/lib/error.js:59
                error = new Error(message);
                        ^

Error: Command failed with exit code 1: esmo src/scripts/migrate-and-start.ts
    at makeError (file:///opt/muse/node_modules/execa/lib/error.js:59:11)
    at handlePromise (file:///opt/muse/node_modules/execa/index.js:119:26)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async file:///opt/muse/src/scripts/run-with-database-url.ts:5:3 {
  shortMessage: 'Command failed with exit code 1: esmo src/scripts/migrate-and-start.ts',
  command: 'esmo src/scripts/migrate-and-start.ts',
  escapedCommand: 'esmo "src/scripts/migrate-and-start.ts"',
  exitCode: 1,
  signal: undefined,
  signalDescription: undefined,
  stdout: undefined,
  stderr: undefined,
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}
```

**Runtime**
I'm running Muse:
- [x] Directly from the cloned repository
- [ ] Inside a Docker container
- [ ] Something else (please elaborate)

**Versions**
- Muse: latest commit
- Docker (if applicable):
- OS: Raspberry Pi OS Lite (64 bit)

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - `/move` command now shows the track that was moved and its position
 
+### Fixed
+- Fixed a case-sensitive import issue
+
 ## [1.8.2] - 2022-03-27
 ### Fixed
 - `/fseek` now works again

--- a/package.json
+++ b/package.json
@@ -109,7 +109,7 @@
     "spotify-uri": "^2.2.0",
     "spotify-web-api-node": "^5.0.2",
     "xbytes": "^1.7.0",
-    "youtube.ts": "^0.2.5",
+    "youtube.ts": "^0.2.6",
     "ytdl-core": "^4.11.0",
     "ytsr": "^3.6.0"
   }

--- a/package.json
+++ b/package.json
@@ -109,7 +109,7 @@
     "spotify-uri": "^2.2.0",
     "spotify-web-api-node": "^5.0.2",
     "xbytes": "^1.7.0",
-    "youtube.ts": "^0.2.6",
+    "youtube.ts": "^0.2.8",
     "ytdl-core": "^4.11.0",
     "ytsr": "^3.6.0"
   }

--- a/src/services/third-party.ts
+++ b/src/services/third-party.ts
@@ -1,6 +1,6 @@
 import {inject, injectable} from 'inversify';
 import SpotifyWebApi from 'spotify-web-api-node';
-import Youtube from 'youtube.ts/dist/Youtube.js';
+import Youtube from 'youtube.ts';
 import {TYPES} from '../types.js';
 import Config from './config.js';
 

--- a/src/services/third-party.ts
+++ b/src/services/third-party.ts
@@ -1,6 +1,6 @@
 import {inject, injectable} from 'inversify';
 import SpotifyWebApi from 'spotify-web-api-node';
-import Youtube from 'youtube.ts/dist/youtube.js';
+import Youtube from 'youtube.ts/dist/Youtube.js';
 import {TYPES} from '../types.js';
 import Config from './config.js';
 

